packages/valory/connections/ledger/ledger_dispatcher.py

```diff
@@ -452,3 +452,12 @@ def get_error_message(
             ),
         )
         return response
+
+    def get_chain_id(self, message: Message) -> str:
+        """
+        Get the chain id. For ledger messages this is the same as the ledger id, for now.
```

Comment:
Would be a good idea to have this be the same as in the contract_dispatcher. I.e. default to leder_id, but allow for it to be overriden. In the keeper we didnt need this, which why we didnt have it.

Reply:
+1